content/blog/page-view-counter/index.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Using Azure Functions to create a page view counter
+date: "2019-12-26T22:12:03.284Z"
+author: nacho
+description:
+  "We can extend static sites by creating a simple page view counter that is processed serverless, and for free. Let's try to build it on Microsoft Azure services."
```

Comment:
I would probably add more things that are SEO friendly in here. The content in itself is fine but we should add a bit more details focusing on using SEO keywords such as 
- Azure (✔️ )
- c# or csharp, the specific azure service you are using (i think is function)
- the general concepts encapsulating all these: `Function as a service` or `Faas` and `Serverless Architecture`
- and any other concept in the post that can help this post being discovered by search engines